gemfiles/rails32.gemfile

```diff
@@ -1,7 +1,6 @@
 source "http://rubygems.org"
 
 gem "rails", ["~> 3.2.6", "!= 3.2.22.1"]
-gem "rack-cache", "~> 1.2.0" # Ruby 1.8.7 requires this
 gem "i18n", "~> 0.6.0"
```

Comment:
This was added in #273, with the note:

> Fix Rails 3.2 gemfile for Ruby 1.8

Looks like it can be dropped here too.